src/rdkafka.h

```diff
@@ -5993,6 +5993,26 @@ typedef rd_kafka_resp_err_t(rd_kafka_interceptor_f_on_thread_exit_t)(
     void *ic_opaque);
 
 
+/**
+ * @brief on_broker_state_chaned() is called just after a broker
+ *        has been created or its state has been changed.
+ *
+ * @param rk The client instance.
+ * @param secproto The security protocol.
+ * @param name The original name of the broker.
+ * @param port The port of the broker.
+ * @param ic_opaque The interceptor's opaque pointer specified in ..add..().
+ *
+ * @returns an error code on failure, the error is logged but otherwise ignored.
+ */
+typedef rd_kafka_resp_err_t(rd_kafka_interceptor_f_on_broker_state_changed_t)(
```

Comment:
this should be named ..._change rather than .._changed to be consistent with the other interceptor naming (e.g., destroy instead of destroyed).